<ui version="4.0" >
 <class>EPGDlg</class>
 <widget class="QDialog" name="EPGDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>422</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor" >
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle" >
   <string>Programme Guide</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="res/icons.qrc" >
    <normaloff>:/icons/MainIcon.png</normaloff>:/icons/MainIcon.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QTabWidget" name="pages" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="GuideTab" >
      <attribute name="title" >
       <string>Guide</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTableView" name="tableView" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="BasicTab" >
      <attribute name="title" >
       <string>Basic Profile Data</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextBrowser" name="basic" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="AdvancedTab" >
      <attribute name="title" >
       <string>Advanced Profile Data</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>6</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextBrowser" name="advanced" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="l1" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonClearCache" >
       <property name="text" >
        <string>Clear Cache</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave" >
       <property name="text" >
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="Spacer2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="l2" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="channel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize" >
           <size>
            <width>300</width>
            <height>32767</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="Spacer4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPrev" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="res/icons.qrc" >
            <normaloff>:/icons/StepBack.png</normaloff>:/icons/StepBack.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonNext" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="res/icons.qrc" >
            <normaloff>:/icons/StepForw.png</normaloff>:/icons/StepForw.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="Spacer4_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="TextEPGDisabled" >
          <property name="text" >
           <string>EPG decoding is disabled</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="Spacer3" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pages</tabstop>
  <tabstop>channel</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="res/icons.qrc" />
 </resources>
 <connections/>
</ui>
